src/components/LocationSelector/CIS/CISCountriesSelect.tsx

```diff
@@ -0,0 +1,169 @@
+import { ReactElement } from 'react';
+import { setCookie } from 'cookies-next';
+import { useSetSelectedEntity } from '../../../types/useSetSelectedEntity';
+import { Cookies, SEVEN_DAYS } from '../../../constants';
+import { LocationSelectorScroll } from '../../LocationSelectorScroll';
+
+interface CISCountriesSelectProps {
+  cityValue?: string;
+}
+
+export const cisRegion: Array<{
```

Comment:
I would move this object to the API or to just *.constants.ts file

Reply:
moved to API, including Russian region